docs/src/man/basics.md

```diff
@@ -0,0 +1,1388 @@
+   5 │   955     57  female      3  rent     rich             little           ⋯
+                                                               3 columns omitted
+```
+
+Equivalently, the `in` function can be called with a single argument to create a function object that
+tests whether each value belongs to the subset (partial application of `in`):
+`german[in(Set([1, 6, 908, 955])).(german.id), :]`
+
+## Not, Between, Cols, findall and All selectors
+
+Finally, you can use `Not`, `Between`, `Cols`, and `All` selectors in more complex column selection
+scenarioes (note that `Cols()` selects no columns while `All()` selects all columns). Now we will 
+see how `Cols` and `Between` can be used to select columns of a data frame. `All()` allows us to select  
+all columns of `DataFrame` while `Between` selector allow us to specify a range of columns (we can 
+specify the start and stop column using any of the single column selector syntaxes). On the other hand, 
+`Not` selector allow us to specify the columns we want to exclude from the resulting data frames. We 
+can put any valid other column selector inside `Not`. Finally `Cols()` selector picks a union of other  
+selectors passed as its arguments.
+
+A `Not` selector (from the [InvertedIndices](https://github.com/mbauman/InvertedIndices.jl) package) 
+can be used to select all columns excluding a specific subset:
+
+```jldoctest dataframe
+julia> german[:, Not(:Age)] # `:Age` was after `:id`
+1000×9 DataFrame
+  Row │ id     Sex     Job    Housing  Saving accounts  Checking account  Cred ⋯
+      │ Int64  String  Int64  String   String           String            Int6 ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     1  male        2  own      NA               little                 ⋯
+    2 │     1  female      2  own      little           moderate
+    3 │     2  male        1  own      little           NA
+    4 │     3  male        2  free     little           little
+    5 │     4  male        2  free     little           little                 ⋯
+    6 │     5  male        1  free     NA               NA
+    7 │     6  male        2  own      quite rich       NA
+    8 │     7  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮       ⋮       ⋮            ⋮                ⋮               ⋱
+  994 │   993  male        3  own      little           little                 ⋯
+  995 │   994  male        2  own      NA               NA
+  996 │   995  female      1  own      little           NA
+  997 │   996  male        3  own      little           little
+  998 │   997  male        2  own      little           NA                     ⋯
+  999 │   998  male        2  free     little           little
+ 1000 │   999  male        2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+```
+
+```jldoctest dataframe
```

Comment:
some comment is lacking

Reply:
comment is still lacking - we should use markdown for commenting not comments in the code if possible